src/test/java/org/openstreetmap/atlas/utilities/caching/CachingTests.java

```diff
@@ -101,85 +49,37 @@ public void testBaseCacheWithNoStrategy()
     public void testLocalFileInMemoryCache()
     {
         final LocalFileInMemoryCache cache = new LocalFileInMemoryCache();
+        URI resourceUri;
+        try
+        {
+            resourceUri = CachingTests.class.getResource(LOG4J_PROPERTIES).toURI();
```

Comment:
Is log4j.properties used as a random file to test the cache, or did you need it for configuring log4j here?

Reply:
It's just a random, non-empty file to test the caching. Should I change it? (I assume it may have logging implications.)